cryptofeed/exchanges/bybit.py

```diff
@@ -38,7 +38,6 @@ class Bybit(Feed):
         LIQUIDATIONS: 'liquidation'
     }
     websocket_endpoints = [
-        WebsocketEndpoint('wss://stream.bybit.com/realtime', channel_filter=(websocket_channels[L2_BOOK], websocket_channels[TRADES], websocket_channels[INDEX], websocket_channels[OPEN_INTEREST], websocket_channels[FUNDING], websocket_channels[CANDLES], websocket_channels[LIQUIDATIONS]), instrument_filter=('QUOTE', ('USD',)), sandbox='wss://stream-testnet.bybit.com/realtime', options={'compression': None}),
```

Comment:
Not sure why this was removed? This would effectively prevent subscribing to the USD trading pairs?

Reply:
I saw your note that it was deprecated, but I dont think thats the case. The `/reatime` endpoint is for the inverse perpetuals: https://bybit-exchange.github.io/docs/inverse/#t-websocket

Reply:
I had misunderstanding about it. 🥲
So, I reverted it.